src/openssl/lib_ssl.cr

```diff
@@ -2,6 +2,9 @@ require "./lib_crypto"
 
 @[Link(ldflags: "`command -v pkg-config > /dev/null && pkg-config --libs libssl || printf %s '-lssl -lcrypto'`")]
 lib LibSSL
+  OPENSSL_102 = {% `command -v pkg-config > /dev/null && pkg-config --atleast-version=102 libssl && printf %s true` == "true" %}
+  OPENSSL_110 = {% `command -v pkg-config > /dev/null && pkg-config --atleast-version=110 libssl && printf %s true` == "true" %}
```

Comment:
I chose to have greater-than / lower-than triggers, which won't break on a new OpenSSL version (unless they removed symbols).